fvm/environment/accounts_status.go

```diff
@@ -10,20 +10,31 @@ import (
 )
 
 const (
-	flagSize            = 1
-	storageUsedSize     = 8
-	storageIndexSize    = 8
-	publicKeyCountsSize = 8
-
-	accountStatusSize = flagSize +
+	flagSize             = 1
+	storageUsedSize      = 8
+	storageIndexSize     = 8
+	publicKeyCountsSize  = 8
+	addressIdCounterSize = 8
```

Comment:
nit: maybe remove `address` prefix for consistency.

```suggestion
	idCounterSize = 8
```

Reply:
It's per address and might be confused with the gloabl UUID counter, I'd recommend to keep the prefix to avoid the confusion